lua/rip-substitute/config.lua

```diff
@@ -36,6 +36,8 @@ local defaultConfig = {
 		},
 	},
 	regexOptions = {
+		-- Start with regex off
+		literalByDefault = false,
```

Comment:
More verbose would be "startWithFixedStringsOn", renounces the need for the comment